src/select/e2e.js

```diff
@@ -0,0 +1,151 @@
+    goToUrl({
+      suite,
+      test: scenarios.SELECT,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .click(selectors.selectInput)
+      .waitForElementPresent(selectors.selectDropDown, 1000);
+  });
+
+  it('opened dropdown can be closed with ESC', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.SELECT,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .click(selectors.selectInput)
+      .waitForElementPresent(selectors.selectDropDown, 1000)
+      .keys('\uE00C')
+      .waitForElementNotPresent(selectors.selectDropDown, 1000);
+  });
+
+  it('selects option when clicked in dropdown', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.SELECT,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .click(selectors.selectInput)
+      .waitForElementPresent(selectors.selectDropDown, 1000)
+      .click(
+        `${selectors.selectDropDown} ${selectors.dropDownOption}:first-child`,
+      )
+      .waitForElementNotPresent(selectors.selectDropDown, 1000)
+      .getText(
+        `${selectors.selectedList} ${selectors.selectedOption}:first-child`,
+        function(result) {
+          this.assert.equal(result.value, 'AliceBlue');
+        },
+      );
+  });
+
+  it('doesnt close dropdown after multiple selections were made', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.MULTI_SELECT,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .click(selectors.selectInput)
+      .waitForElementPresent(selectors.selectDropDown, 1000)
+      .click(
+        `${selectors.selectDropDown} ${selectors.dropDownOption}:first-child`,
+      )
+      .click(
+        `${selectors.selectDropDown} ${selectors.dropDownOption}:nth-child(3)`,
+      )
+      .waitForElementPresent(selectors.selectDropDown, 1000);
+  });
+
+  it('doesnt allow to click and select disabled options', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.SELECT_DISABLED_OPTIONS,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .click(selectors.selectInput)
+      .waitForElementPresent(selectors.selectDropDown, 1000)
+      .click(
+        `${selectors.selectDropDown} ${selectors.dropDownOption}:nth-child(2)`,
+      )
+      .waitForElementPresent(selectors.selectDropDown, 1000)
+      .getText(selectors.selectedList, function(result) {
+        this.assert.equal(result.value, '');
+      });
+  });
+
+  it('selects options when search input successful with results', browser => {
+    goToUrl({
+      suite,
+      test: scenarios.MULTI_SELECT_SEARCH,
+      browser,
+    })
+      .waitForElementVisible(selectors.selectInput, 1000)
+      .setValue(selectors.selectInput, 'dark', () => {
+        browser
+          .waitForElementPresent(selectors.selectDropDown, 100)
+          .pause(1000)
```

Comment:
Can we increase the wait time to `1000` on the line 139 instead of `pause` here?

Reply:
that will not work as `pause`, which works to wait for async call, which we emulate here. Cause DropDown is shown immediately with loading spinner, but no data yet there

Reply:
Oh, I see. Makes sense. Another approach would be to wait until a specific option is visible. If we keep `pause` versus `wait for an option`, could you add a comment to it about the async call and a loading state here?